operations/observability/mixins/platform/rules/scrape.libsonnet

```diff
@@ -0,0 +1,29 @@
+/**
+ * Copyright (c) 2022 Gitpod GmbH. All rights reserved.
+ * Licensed under the MIT License. See License-MIT.txt in the project root for license information.
+ */
+
+{
+  prometheusAlerts+:: {
+    groups+: [
+      {
+        name: 'prometheus-scraping-rules',
+        rules: [
+          {
+            alert: 'TargetDownOrFailedScrape',
+            labels: {
+              severity: 'warning',
+              team: 'platform',
+            },
+            'for': '10m',
+            annotations: {
+              summary: 'Prometheus failed to scrape {{ $labels.job }}',
+              description: 'Prometheus couldn\'t scrape {{ printf "%.4g" $value }}% of the {{ $labels.job }} targets. Components could be unnavailable or we have some scraping misconfiguration.',
+            },
+            expr: '100 * (count(up{container!="workspace"} == 0) BY (job) / count(up{container!="workspace"}) BY (job)) > 10',
```

Comment:
```suggestion
            expr: '(count(up{container!="workspace"} == 0) BY (job) / count(up{container!="workspace"}) BY (job)) > 0',
```

Reply:
Ah, makes sense the 0, but 100 * is there so we have a good message in slack 😛 
`Prometheus couldn't scrape {{ printf "%.4g" $value }}%....`